config/lib/socket.io.js

```diff
@@ -21,7 +21,34 @@ module.exports = function (app, db) {
     var certificate = fs.readFileSync('./config/sslcerts/cert.pem', 'utf8');
     var options = {
       key: privateKey,
-      cert: certificate
+      cert: certificate,
+      //  requestCert : true,
+      //  rejectUnauthorized : true,
+      secureProtocol: 'TLSv1_method',
+      ciphers: [
```

Comment:
@lirantal I got it from the nodejs source code

https://github.com/nodejs/node/blob/master/lib/tls.js

Once mean.js is up on the latest node version, we can remove this list of ciphers or update it based on the latest security research.